### ISSUE TITLE
irb and rails console broken on windows 10

### ISSUE THREAD

**Issue by cshupp1** [jruby/jruby#5534]
If I bring up an irb session as follows:

```
java -jar C:\Users\cshupp\Downloads\jruby-complete-9.2.5.0.jar -Sjirb
```
The up, down arrows for history do not work.  The left and right arrows also do not work.

Bringing it up this way:
```
java -jar C:\Users\cshupp\Downloads\jruby-complete-9.1.10.0.jar -Sjirb
```
They do work. 

Bringing the latest version up in a Cygwin shell is a workaround on later versions of JRuby.

**Comment by kimptoc** [comment]
I think its been broken since 9.2.2 at least - still fails in 9.2.5 and 9.2.6.  Works in 9.2.0.0 for me. But I have to explode the jar file and run it on the classpath (using the jar does not work).  also using java 11 :)

```
java -cp jruby-complete-9.2.0.0 org.jruby.Main -Sjirb 
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.jruby.runtime.encoding.EncodingService (file:/Z:/scripts/tools/jruby/jruby-complete-9.2.0.0/) to field java.io.Console.cs
WARNING: Please consider reporting this to the maintainers of org.jruby.runtime.encoding.EncodingService
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```

**Comment by headius** [comment]
Confirmed by @enebo as still broken. We will look to fix this for 9.3, perhaps by incorporating reline in #5803.

**Comment by enebo** [comment]
This particular report was fixed in 9.4.0.0 with the move to reline.  There was still a problem with require trying to load Win32API.rb in jruby-complete but that was fixed by renaming the file to win32api.rb.  This is fixed.
